algorithms/baselines/ppo/agent_ppo.py

```diff
@@ -212,6 +238,20 @@ def __init__(self, make_env_func, params):
         all_vars = tf.trainable_variables()
         slim.model_analyzer.analyze_vars(all_vars, print_info=True)
 
+    def input_dict(self, buffer, start, end):
```

Comment:
Here you are feeding advantages and action probes that critic does not need, but I think it's okay. This way code is cleaner.

Reply:
Yes I thought about that before feeding the same dictionary to both actor and critic